modules/tinymce/src/models/dom/main/ts/table/Table.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { SugarElement } from '@ephox/sugar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+import * as Clipboard from './actions/Clipboard';
+import { TableActions } from './actions/TableActions';
+import { Api, getApi } from './api/Api';
+import { Clipboard as FakeClipboard } from './api/Clipboard';
+import * as Commands from './api/Commands';
+// import * as Options from './api/Options';
+import * as QueryCommands from './api/QueryCommands';
+import { ephemera } from './selection/Ephemera';
+
+export interface PatchedSelections {
+  readonly get: () => SugarElement<HTMLTableCellElement>[];
+}
+
+const setupTable = (editor: Editor): Api => {
+  // Options.register(editor);
```

Comment:
seems like some commented out code that can be deleted (here and in `./api/Options.ts`)

Reply:
Have removed the commented Options code